drivers/i3c/i3c_stm32.c

```diff
@@ -474,9 +474,11 @@ static int i3c_stm32_calc_scll_od_sclh_i2c(const struct device *dev, uint32_t i2
 				   1;
 			*sclh_i2c = DIV_ROUND_UP(i3c_clock, i2c_bus_freq) - *scll_od - 2;
 			if (*sclh_i2c <
-				DIV_ROUND_UP(STM32_I3C_SCLH_I2C_MIN_FM_NS * i3c_clock, 1000000000ull) - 1) {
-				LOG_ERR("Cannot find a combination of SCLL_OD and SCLH_I2C at current I3C "
-					"clock "
+				DIV_ROUND_UP(
+					STM32_I3C_SCLH_I2C_MIN_FM_NS * i3c_clock, 1000000000ull
+				) - 1) {
+				LOG_ERR("Cannot find a combination of SCLL_OD and SCLH_I2C at "
+					"current I3C clock "
 					"frequency for FM I2C bus");
```

Comment:
Following looks better
```suggestion
				LOG_ERR("Cannot find a combination of SCLL_OD and SCLH_I2C at "
					"current I3C clock frequency for FM I2C bus");
```

Reply:
Not resolved

Reply:
This would still require a fix. Last change and you're done